src/components/canvas.js

```diff
@@ -39,11 +39,27 @@ const canvasView = widget((update) => {
     </div>
   `
 
+  function renderCanvas () {
```

Comment:
I changed this because the variables are accessed through the scope anyways and not passed to the functions which calls renderCanvas. So it felt like unnecessary duplication.